### ISSUE TITLE
HomeKit config randomly lost

### ISSUE THREAD

**Issue by fredrikbae** [swttt/com.swttt.homekit#128]
HomeKit (on the Apple side) randomly looses room and automation config of HomeyKit devices. It seems like HomeyKit stops presenting the devices just long enough for HomeKit to forget them and then starts presenting them again.

**Comment by robertklep** [comment]
Strange :( I don't use any automations, but I've never lost room configuration.

Have you previously used the built-in experimental Homekit support from Homey, or is that still active?

**Comment by fredrikbae** [comment]
Yes, after the upgrade to 2.0 i first tried the built-in HomeKit-support as the the other app I used stoped working.  Both the built in support and the other app are disabled/uninstalled both in Homey and on the Apple-side.

Rgds Fredrik

Fredrik Baecklund
+4670-4247347
________________________________
Från: Robert Klep <notifications@github.com>
Skickat: tisdag, januari 29, 2019 11:07 fm
Till: swttt/com.swttt.homekit
Kopia: Fredrik Baecklund; Author
Ämne: Re: [swttt/com.swttt.homekit] HomeKit config randomly lost (#128)


Strange :( I don't use any automations, but I've never lost room configuration.

Have you previously used the built-in experimental Homekit support from Homey, or is that still active?

**Comment by robertklep** [comment]
Tracking in #131
